base/python/R/R_nonstandard/rstudio_shiny/Dockerfile

```diff
@@ -2,21 +2,56 @@ FROM cityofcapetown/datascience:R_nonstandard
 
 LABEL authors="Riaz Arbi,Gordon Inggs"
 
-# Install rstudio
-COPY rstudio_additions.sh .
-RUN bash rstudio_additions.sh
+# Install and setup Pandoc
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
```

Comment:
I subsequently did this in a commit to `staged_builds`. 

Please rebase against it - based on the merge conflicts, it should be reasonably seamless.